Dockerfile

```diff
@@ -49,8 +51,7 @@ RUN cp .github/docker/supervisord.conf /etc/supervisord.conf && \
 HEALTHCHECK --interval=5m --timeout=10s --start-period=5s --retries=3 \
   CMD curl -f http://localhost/up || exit 1
 
-EXPOSE 80:2019
-EXPOSE 443
+EXPOSE 80 443
```

Comment:
If you run caddy in http it defaults to 2019. I have no idea why but it does.

Reply:
Maybe cause of [this](https://caddyserver.com/docs/caddyfile/options#admin) ?

Reply:
I meant that I don't know why they defaulted to 2019. They could have used 80 and had been fine.